datalake/backend/tests/test_import_dataset_logging.py

```diff
@@ -0,0 +1,126 @@
+import json
+import logging
+from json import dumps
+from unittest.mock import MagicMock, patch
+
+from jsonschema import ValidationError  # type: ignore[import]
+from pytest import mark
+from pytest_subtests import SubTests  # type: ignore[import]
+
+from ..processing.import_dataset.task import lambda_handler
+from .utils import (
+    ProcessingAsset,
+    any_dataset_id,
+    any_dataset_version_id,
+    any_hex_multihash,
+    any_lambda_context,
+    any_s3_url,
+)
+
+
+class TestLogging:
+    logger: logging.Logger
+
+    @classmethod
+    def setup_class(cls) -> None:
+        cls.logger = logging.getLogger("datalake.backend.processing.import_dataset.task")
+
+    @mark.infrastructure
+    @patch("datalake.backend.processing.import_dataset.task.s3control_client.create_job")
+    def test_should_log_payload(self, create_job_mock: MagicMock) -> None:
+        # Given
+
+        create_job_mock.return_value = {"JobId": "Some Response"}
+
+        body = {}
+        body["dataset_id"] = any_dataset_id()
+        body["metadata_url"] = any_s3_url()
+        body["version_id"] = any_dataset_version_id()
+        expected_payload_log = dumps({"payload": body})
+
+        with patch.object(self.logger, "debug") as log_mock:
+
+            # When
+            lambda_handler(body, any_lambda_context())
+
+            # Then
+            log_mock.assert_any_call(expected_payload_log)
+
+    @patch("datalake.backend.processing.import_dataset.task.validate")
+    def test_should_log_missing_argument_warning(self, validate_schema_mock: MagicMock) -> None:
```

Comment:
This title doesn't correspond to the test contents. It looks instead like it logs any validation error.